src/docker/mod.rs

```diff
@@ -31,6 +31,10 @@ impl ProvidedImage {
     pub fn image_name(&self, repository: &str, tag: &str) -> String {
         image_name(self.name, self.sub, repository, tag)
     }
+
+    pub fn default_image_name(&self) -> String {
+        self.image_name(CROSS_IMAGE, DEFAULT_IMAGE_VERSION)
+    }
```

Comment:
is this function really needed? I feel like it could be done inplace.

Reply:
my only concern with it is that in context it's a bit hard to understand what default_image_name actually is.

Reply:
I apologize, I'm not sure I understand what you mean by that. 😅
I think that without it, you'd still have to look at the definition of `image_name` and the two consts, so in that respect I don't think it adds much mental overhead. I'm not sure that's what you meant, though.

Reply:
you're right, lets leave the function!